nginx.conf

```diff
@@ -0,0 +1,42 @@
+user  nginx;
+worker_processes  auto;
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    
+    keepalive_timeout  65;
+
+    gzip  on;
+
+    server {
+        listen       80;
+        listen 443 default_server ssl;
+
+        server_name         github.com;
+        ssl_certificate /etc/ssl/certs/github.com.crt;
+        ssl_certificate_key /etc/ssl/private/github.com.key;
+
+        location /istio/istio/releases/download/ {
+            alias   /opt/istio-releases/;
+        }
```

Comment:
As discussed in other thread, volumes will need to allow users to provide their own manifests.

Reply:
There is no flag or templating needed for this - the volume uses this path to mount the additional tar file to.